services/app-api/handlers/users/update.js

```diff
@@ -7,33 +7,30 @@ export const main = handler(async (event, context) => {
     console.log("Warmed up!");
     return null;
   }
+
   const data = JSON.parse(event.body);
+
   const params = {
-    TableName: process.env.AuthUserTableName,
-    // 'Key' defines the partition key and sort key of the item to be updated
-    // - 'userId': Identity Pool identity id of the authenticated user
-    // - 'amendmentId': path parameter
+    TableName:
+      process.env.AUTH_USER_TABLE_NAME ?? process.env.AuthUserTableName,
     Key: {
       userId: data.userId,
     },
-    // 'UpdateExpression' defines the attributes to be updated
-    // 'ExpressionAttributeValues' defines the value in the update expression
-    UpdateExpression: "SET #r = :role, states = :states, isActive = :isActive",
+    UpdateExpression:
+      "SET #r = :role, states = :states, isActive = :isActive, lastLogin = :lastLogin",
     ExpressionAttributeValues: {
       ":role": data.role,
-      ":states": data.states,
+      ":states": data.states ?? "",
```

Comment:
data.states ?? "" whats up with this line and what is the purpose of this

Reply:
If data.states doesn't exist, user an empty string.

Reply:
makes sense. why is that not for the rest of the attribute